src/App.svelte

```diff
@@ -55,9 +56,25 @@
 			'Tab': 'autoIndentMarkdownList',
 			'Shift-Tab': 'autoUnindentMarkdownList'
 		});
-
 		editor.setValue($text);
 		validateText();
+		const urlParams = new URLSearchParams(window.location.search);
+		if(urlParams.has('id')) {
+			const channelsJSON = await rawGithubJSONRequest('https://raw.githubusercontent.com/pvme/pvme-settings/pvme-discord/channels.json');
+			for(const channel of channelsJSON) {
+				if(channel.id === urlParams.get('id')) {
+					//showModal = true;
+					guideUrl = `https://raw.githubusercontent.com/pvme/pvme-guides/master/${channel.path}`;
```

Comment:
move to function like `findGuideUrlFromID()`

Reply:
trying to make a findGuideUrl made it super annoying since I still needed the channel obj for the rest of it. Opted to just move everything into loadGuide hope that's cool